Libraries/Text/Text/RCTTextShadowView.m

```diff
@@ -35,6 +35,11 @@ - (instancetype)initWithBridge:(RCTBridge *)bridge
   return self;
 }
 
+-(void)didSetProps:(NSArray<NSString*>*)changedProps {
```

Comment:
Could you please add a comment here describing why we need this and apply code style?

Reply:
Will do in the morning.